subsys/bluetooth/Kconfig

```diff
@@ -39,6 +39,16 @@ config BT_CUSTOM
 
 endchoice
 
+if !BT_RECV_IS_RX_THREAD
+
+config SYSTEM_WORKQUEUE_STACK_SIZE
+	default 3092 if BT_MESH_GATT_CLIENT
+	default 2200 if BT_SETTINGS
+	default 2048 if BT_MESH
+	default 2048 if BT_AUDIO
+
+endif # !BT_RECV_IS_RX_THREAD
+
 # The Bluetooth subsystem requires the system workqueue to execute at
 # a cooperative priority.
 config SYSTEM_WORKQUEUE_PRIORITY
```

Comment:
@jhedberg @Thalley @nashif 

>  if some third option might exist (or be created).

Refer to how the priority is used, so that it can be automatically configured without polluting the code in Kernel Kconfig, maybe this is the third way you want. :)